.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+    branches: [master]
+  schedule:
+    - cron: '0 6 * * 1'  # Every monday 6 AM
```

Comment:
Just wondering, is it a good idea to schedule regular builds for every repo at the same time?

Reply:
Yes, it's a good idea :)  Notifications, if any, go in all at once.

[I set *all* my personal repos to the same time](https://github.com/search?q=user%3Aoprypin+%22cron%3A+%270+6+*+*+6%27%22+language%3Ayaml&type=code) and it's fine.